views/form.pug

```diff
@@ -0,0 +1,8 @@
+form(action='http://localhost:3000/auth/google', method='POST')
```

Comment:
do czego służą inputy dla username i password? Jeśli używasz oAuth2 to takie dane są zbędne. W dokumentacji passporta możesz użyć czegoś takiego: `<a href="/auth/google">Sign In with Google</a>`